newm-server/src/main/resources/application.conf

```diff
@@ -287,6 +287,7 @@ marketplace {
     enabled = true
     enabled = ${?MARKETPLACE_MONITOR_DAEMON_ENABLED}
     pointerPolicyId = ${AWS_SECRET_ARN}
+    pendingOrderTimeToLive = 60
```

Comment:
This stuff should all go into the config repository instead of environment configs.